Kernel/API/Syscall.h

```diff
@@ -19,6 +20,7 @@ struct sockaddr;
 struct siginfo;
 struct stat;
 struct statvfs;
+struct sched_param;
```

Comment:
Do we require that definition? We already pulled in `Kernel/API/POSIX/sched.h` earlier.